Sum of squares of N Natural numbers added( #1304)

shonali2600 opened the pull request:
fixes #1304

sairish2001 reviewed (changes requested):
I think by mistake, you deleted the Nth term of HP button

shonali2600 commented:
Actually I misunderstood the `nth term of HP` part of code with `sum of n terms of HP` and thought that it was repeated so I deleted that. Now resolved and I added the comments for the same.
